app/controllers/admin/users_controller.rb

```diff
@@ -118,6 +142,7 @@ def title_for_index
     private def adding_admin?
       @adming_admin ||= begin # rubocop:disable Naming/MemoizedInstanceVariableName
```

Comment:
Unrelated, but..
```suggestion
      @adding_admin ||= begin
```

Reply:
I believe that's necessary because the method hast a question mark, which sadly isn't quite the same as @adding_admin.